Tests.Unit.Presentation.Web/Controllers/CronStringHelperTests.cs

```diff
@@ -21,6 +21,9 @@ public class CronStringHelperTests
         [Theory]
         [InlineData(Scheduling.Quarter, 2000, 1, 1, "0 8 1 1-12/3 *")]
         [InlineData(Scheduling.Quarter, 1999, 11, 29, "0 8 29 2-12/3 *")]
+        [InlineData(Scheduling.Month, 1999, 12, 31, "0 8 L * *")]
```

Comment:
Hvad med en test af februar på skudår? Eller kan DateTime ikke håndtere det?

Reply:
jo det kan den uden tvivl. Jeg tilføjer en test med forskellige vinkler

Reply:
Cronstringhelper er nok mest den "dumme" "oversætter". Den kommer ikke til at lave alle tricks, for man skal bruge advicetriggerfactory. Så vil den forholde sig til februar. Til gengæld blev jeg obs på at "årlig" med 29. februar ikke blev håndteret i advicetriggerfactory og det er jo netop også en special case som skal oversættes til "0 8 L 2 *"